tests/libnet/namespace.go

```diff
@@ -0,0 +1,99 @@
+package libnet
+
+import (
+	"context"
+	"encoding/json"
+
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/strategicpatch"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func AddLabelToNamespace(namespace, key, value string) {
```

Comment:
First of, thanks for exporting these.

Despite that, there's one thing I don't like - the expects inside this helper. I think you should return the error, and the caller is the one that should expect the call to succeed.

Reply:
Thanks, I agree, I forgot to update these appropriately. Done